pkg/controllers/user/secret/secret.go

```diff
@@ -66,19 +67,21 @@ func (n *NamespaceController) sync(key string, obj *corev1.Namespace) error {
 	if obj.Annotations[projectIDLabel] != "" {
 		parts := strings.Split(obj.Annotations[projectIDLabel], ":")
 		if len(parts) == 2 {
-			// on the managemenet side, secret's namespace name equals to project name
+			// on the management side, secret's namespace name equals to project name
 			secrets, err := n.managementSecrets.List(parts[1], labels.NewSelector())
 			if err != nil {
 				return err
 			}
 			for _, secret := range secrets {
 				namespacedSecret := &corev1.Secret{}
```

Comment:
can we extract the method for setting up the secret fields? Pass secret you are copying from, as a reference.